docs/axes/labelling.md

```diff
@@ -55,6 +55,13 @@ const chart = new Chart(ctx, {
 });
 ```
 
+Keep in mind that overriding `ticks.callback` means that you are responsible for all formatting of the label. Depending on your use case, you may want to call the default formatter and then modify its output. In the example above, that would look like:
+
+```javascript
+                        // call the default formatter, forwarding `this`
```

Comment:
It looks like there's some leading whitespace here which could be removed

Reply:
I kept the leading whitespace the same as in the example above, to emphasize that the snippet would substitute for `return '$' + value;`.